### PR TITLE
[New Rule] Add Living-off-the-Land (LotL) ProblemChild Rules

### PR DESCRIPTION
**Pull request by terrancedejesus**
## Issues
* https://github.com/elastic/ia-trade-team/issues/190

## Summary
This PR adds detection rules for the LotL analytic package using the ProblemChild algorithm.
Unit tests will fail until https://github.com/elastic/detection-rules/pull/3102/ is merged.

### CODE REVIEW

**Review by eric-forte-elastic — APPROVED** [review]
:green_circle: Manual review, looks good to me! :+1:

Very nice ML rules :rocket: !

File: rules/integrations/problemchild/defense_evasion_ml_suspicious_windows_process_cluster_from_parent_process.toml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    "https://docs.elastic.co/en/integrations/problemchild"
+]
+risk_score = 21
+rule_id = "f5d9d36d-7c30-4cdb-a856-9f653c13d4e0"
+severity = "low"
+tags = [
+    "Use Case: Living off the Land Attack Detection",
+    "Rule Type: ML",
+    "Rule Type: Machine Learning",
+    "Tactic: Defense Evasion",
```
Comment (w0rk3r) [inline rules/integrations/problemchild/defense_evasion_ml_suspicious_windows_process_cluster_from_parent_process.toml:40]:
```suggestion
    "Domain: Endpoint",
    "OS: Windows",
    "Use Case: Living off the Land Attack Detection",
    "Rule Type: ML",
    "Rule Type: Machine Learning",
    "Tactic: Defense Evasion",
```

**Review by w0rk3r — COMMENTED** [review]
Left some comments/suggestions

**Review by Mikaayenson — COMMENTED** [review]
In general LGTM with @w0rk3r updates and the reference to setup instructions.

**Review by ajosh0504 — APPROVED** [review]
Rules looking good! Double-checked that min stack version is 8.9.0. 🚀 

@terrancedejesus Thanks for adding the setup instructions and additional references! ✨

**Review by Mikaayenson — APPROVED** [review]

**Pull request merged**